lib/instance/instance_view.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:lemmy_api_client/v3.dart';
+import 'package:thunder/shared/common_markdown_body.dart';
+
+class InstanceView extends StatelessWidget {
+  final Site site;
+  final String? alternateSiteName;

```

Comment:
Out of curiosity, why is this needed?

Reply:
I was trying to interpret some strange logic in the existing community sidebar.

https://github.com/thunder-app/thunder/blob/1047f0663eff97368a7da247634a712f605106d3/lib/community/widgets/community_sidebar.dart#L144-L150

I think I interpreted this as using one of the moderator names as an alternate name for the community if needed, but I think it's actually just using the first letter as the icon if there is no icon. This is still strange to me.

I'm going to take this out and use the first letter of the community name instead!